docs/eventing/experimental-features/sender-identity.md

```diff
@@ -0,0 +1,233 @@
+# Sender Identity for Knative Eventing
+
+**Flag name**: `authentication-oidc`
+
+**Stage**: Alpha, disabled by default
+
+**Tracking issue**: [#6806](https://github.com/knative/eventing/issues/6806)
+
+## Overview
+
+Currently, event delivery within the cluster is unauthenticated, and an addressable event consumer cannot determine the identity of any sender.
+
+Knative Eventing Addressables expose their OIDC Audience in their status as part of their address (e.g. `.status.address.audience`) and require requests containing an OIDC access token issued for this audience.
+
+Knative Eventing Souces request an OIDC access token for the targets audience and add them to the request. A per-source dedicated Service Account is used as the identity for the request.
+
+## Prerequisites
+
+- [Eventing installation](./../../install)
```

Comment:
The link here is invalid. I would suggest use the absolute path

Reply:
Good catch. Updated it